backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -241,6 +268,17 @@ def frame_export_json(
             llm = ""
             embedding_model = ""
 
+        if invalidated_prompts:
+            raise InValidCustomToolError(
+                f"Cannot export tool. Prompt(s) : {invalidated_prompts} "
+                "do not have a valid prompt."
+            )
+        if invalidated_outputs:
+            raise InValidCustomToolError(
+                f"Cannot export tool. Prompt(s) : {invalidated_outputs} "
+                "are not executed."
```

Comment:
NIT: Similar pretty print of lists. Also a suggestive action can be added here to make it
```
Cannot export Tool. Prompt(s): 'Test_prompt_1', 'Test_prompt_2' were not run. Please run them first
``` 

The intention is to stick with what we refer to in the product. For example - I believe though we call it as "prompt execution" we refer to it as "Run Prompt" in the UI so it'll be good to follow the same name in errors